deploy/manifests/prod/us-east-2/cluster/kube-system/aws-auth.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: aws-auth
+  namespace: kube-system
+data:
+  mapRoles: |
+    - groups:
+      - system:bootstrappers
+      - system:nodes
+      rolearn: arn:aws:iam::407967248065:role/prod-ue2-r5b-xl-eks-node-group
+      username: system:node:{{EC2PrivateDNSName}}
+    - groups:
+      - system:bootstrappers
+      - system:nodes
+      rolearn: arn:aws:iam::407967248065:role/prod-ue2-m4-xl-eks-node-group
+      username: system:node:{{EC2PrivateDNSName}}
+  mapUsers: |
```

Comment:
can we have access templated to be shared between deployments, rather than duplicated in each instance / zone?

Reply:
I _think_ we should be able to for users 👍 I will address this in a separate PR.